Spec/RealtimeClientPresence.swift

```diff
@@ -904,7 +904,7 @@ class RealtimeClientPresence: QuickSpec {
                             guard let error = error else {
                                 fail("error expected"); done(); return
                             }
-                            expect(error.message).to(contain("Channel denied access based on given capability"))
+                            expect(error.message).to(contain("Invalid clientId for credentials"))
```

Comment:
I looked at the lib and I found nothing that could change this behaviour. It seems to be an Ably service change. @tcard what do you think?